demo/js/demo.tsx

```diff
@@ -89,9 +89,9 @@ const Example = (): JSX.Element => (
                         </thead>
                         <tbody>
                             <tr>
-                                <td>accordion</td>
+                                <td>allowMultipleExpanded</td>
                                 <td>Boolean</td>
-                                <td>true</td>
+                                <td>false</td>
                                 <td>Open only one item at a time or not</td>
```

Comment:
I think this language should now be more explicit: "Don't close all the others when expanding an AccordionItem" or similar.